src/Integration.UnitTests/Binding/DotNetBindingConfigProviderTests.cs

```diff
@@ -151,7 +151,7 @@ public async Task GetConfig_NuGetBindingOperationFails_ReturnsNull()
         }
 
         [TestMethod]
-        public async Task GetConfig_HasActiveAndInactiveRules_ReturnsValidBindingConfig()
+        public async Task GetConfig_HasActiveInactiveAndUnsupportedRules_ReturnsValidBindingConfig()
```

Comment:
Modified the existing full generation test to check that it is calling the rule filter method.